api/HTMLFormElement.json

```diff
@@ -91,7 +91,7 @@
               "version_added": null
             },
             "webview_android": {
-              "version_added": null
+              "version_added": true
```

Comment:
This is https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/forms/html_form_element.idl, old stuff that's definitely in WebView.